Makefile

```diff
@@ -34,7 +35,6 @@ deb: build
 	-t deb \
 	-v $(VERSION) \
 	--deb-no-default-config-files \
-	--deb-use-file-permissions \
```

Comment:
Mind trying to understand the issue, perhaps opening an Issue in fpm's repo
if need be?